networking/udp/udp_socket/runtest.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved.
+#
+# This copyrighted material is made available to anyone wishing
+# to use, modify, copy, or redistribute it subject to the terms
+# and conditions of the GNU General Public License version 2.
+#
+# This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+source ../../common/include.sh || exit 1
+
+iproute2_install
```

Comment:
Hi @idorax I checked the last Beaker job (3540924) and the taskout log contains

```./runtest: line 22: iproute2_install: command not found```

Can you take a look what happened? There's either a problem with includes or we shouldn't be calling a nonexistent command

Reply:
Probably because the library was never extracted properly, see my comment above.

Reply:
Hi @veruu and @rasibley, `iproute2_install` is a function which is not included in file `networking/common/install.sh` ([1])

- [1] https://github.com/CKI-project/tests-beaker/blob/master/networking/common/install.sh

I removed it because it seems useless when testing RHEL7/8/ark/...
```Bash
# install ip netns support on RHEL6
iproute2_install()
{
	{ [ "$(GetDistroRelease)" -eq 7 ] || ip netns; } && return 0
	local OUTPUTFILE=$(new_outputfile)
	local iproute_ver=${IPROUTE_VER:-"2.6.32-130.el6eng.netns.2"}
	local major_ver=$(echo $iproute_ver | cut -d'-' -f1)
	local minor_ver=$(echo $iproute_ver | cut -d'-' -f2)
	local iproute_netns="http://download-node-02....
       ...<snip>...
}
```

Reply:
Fine by me. We will need to keep it in mind if we start running with RHEL6 but no worries for now.